### PR TITLE
Request to add APTS token

### PR DESCRIPTION
**Pull request by business-bluemove**
Hi Hippo Space Team,

The BlueMove team is excited to request the addition of the APTS coin to Aptos-coin-list on GitHub. Recently, BlueMove successfully launched the $APTS inscription token, following the DA standard, and it has been warmly welcomed by the community.

In our efforts to enhance the utility of the $APTS token, we've also developed a bridge that allows seamless conversion between the $APTS inscription token and $APTS coin. For further details, please refer to our announcement here: 
https://twitter.com/BlueMove_OA/status/1736656999621009858

We greatly appreciate your team's efforts and would love to receive your feedback soon.

Thanks and best regards

### CODE REVIEW

File: src/requestList.ts [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+    token_type: {
+      type: "0xc71d94c49826b7d81d740d5bfb80b001a356198ed7b8005ae24ccedff82b299c::bridge::APTS",
+      account_address: "0xc71d94c49826b7d81d740d5bfb80b001a356198ed7b8005ae24ccedff82b299c",
+      module_name: "bridge",
+      struct_name: "APTS"
+    },
+    extensions: {
+      data: [["bridge", "native"]]
+    },
+    unique_index: 928,
```
Comment (movekevin) [inline src/requestList.ts:21]: You'd need a unique value here. This overlaps with the next entry

**Comment by business-bluemove** [comment]
Hi @movekevin 

Could you assist me in adjusting the unique_index value? I want to ensure it aligns with HippoSpace's management requirements. 

Thanks!

**Comment by business-bluemove** [comment]
Hi  @movekevin and the team

Could you please let me know of any updates on this

Thanks!

**Comment by movekevin** [comment]
> Hi @movekevin and the team
> 
> Could you please let me know of any updates on this
> 
> Thanks!

You'll need to find a unique value across the file to use as an id

**Comment by business-bluemove** [comment]
Hi  @movekevin, and the team

I changed the unique value for the APTS coin. Please help me to review it

Thanks!

**Pull request merged**